homeassistant/components/fujitsu_fglair/const.py

```diff
@@ -2,14 +2,14 @@
 
 from datetime import timedelta
 
-from ayla_iot_unofficial.fujitsu_consts import (  # noqa: F401
-    FGLAIR_APP_ID,
-    FGLAIR_APP_SECRET,
-)
+from ayla_iot_unofficial.fujitsu_consts import FGLAIR_APP_CREDENTIALS  # noqa: F401
```

Comment:
why the noqa?

Reply:
Because FGLAIR_APP_CREDENTIALS is imported here just to be exported under the components.fujitsu_fglair namespace so ruff complains about an unused import without the noqa.

Reply:
Why don't we just use it where we need it?